pym/bob/scm/git.py

```diff
@@ -294,7 +294,7 @@ async def __checkoutTag(self, invoker, fetchCmd, switch):
         head = await invoker.callCommand(["git", "rev-parse", "--verify", "-q", "HEAD"],
             stdout=False, cwd=self.__dir)
         if head or switch:
-            await invoker.checkCommand(fetchCmd, cwd=self.__dir)
+            await invoker.checkCommand(fetchCmd, retries=10, cwd=self.__dir)
```

Comment:
I find 10 a little bit over the top. Maybe 3? And what about the other `git fetch` invocations? I guess there we should also retry...

Reply:
Yesterday it failed with 10 retries... :( Maybe this needs to be a scm-property. I also added a sleep(5) after each retry but without success. 🤷

Reply:
Hmm, my assumption is that if it did not work with three retries then it most probably won't work with 10 retries either. BTW, the retries are probably also needed for `ls-remote` calls.

Making the retries configurable sounds like a good idea. I think a global retry count is probably sufficient. If there is ever a need to make it configurable per-SCM, we can still introduce a property...